### PR TITLE
Create OWNERS files for the Framefrogs team

### PR DESCRIPTION
**Pull request by majakurcius**
Create OWNER files for the Framefrogs team in:

- The `application-connector` chart
- The `connector-service` subchart
- The `connection-token-handler` subchart
- The `connectivity-certs-controller` subchart
- The `application-registry` subchart
- The `application-operator` subchart
- The `application-connector-helper` chart
- The `application-connector-ingress` chart
- The `compass` chart
- The `compass-runtime-agent` chart
- Connector Service Component
- Connection Token Handler Component
- Connectivity Certs Controller Component
- Application Registry Component
- Application Operator Component
- Application Gateway Component
- Application Connectivity Validator Component
- Application Connectivity Certs Setup Job Component
- Compass Runtime Agent
- Connector Service Tests
- Application Registry Tests
- Application Gateway Tests
- Application Operator Tests
- Application Connector Tests
- Connection Token Handler Tests
- Compass Runtime Agent Tests

**Related issue(s)**
--

### CODE REVIEW

File: components/compass-runtime-agent/OWNERS [hunk 1]
```diff
@@ -0,0 +1,12 @@
+approvers:
```
Comment (crabtree) [inline components/compass-runtime-agent/OWNERS]: I would move PK85, aszecowka, pkosiec, kfurgol and tgorgol to reviewers section

File: resources/application-connector-helper/OWNERS [hunk 2]
```diff
@@ -0,0 +1,13 @@
+approvers:
```
Comment (crabtree) [inline resources/application-connector-helper/OWNERS]: I would move piotrmsc, kubadz, strekm, jakkab, Tomasz-Smelcerz-SAP and Demonsthere to reviewers section

File: resources/compass-runtime-agent/OWNERS [hunk 3]
```diff
@@ -0,0 +1,12 @@
+approvers:
```
Comment (crabtree) [inline resources/compass-runtime-agent/OWNERS]: Same as `components/compass-runtime-agent/OWNERS`

File: tests/compass-runtime-agent/OWNERS [hunk 4]
```diff
@@ -0,0 +1,12 @@
+approvers:
```
Comment (crabtree) [inline tests/compass-runtime-agent/OWNERS]: Same as `components/compass-runtime-agent/OWNERS`

File: components/compass-runtime-agent/OWNERS [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  - franpog859 
+  - Maladie
+  - crabtree
+
+reviewers:
+  - PK85
+  - aszecowka
+  - pkosiec
+  - kfurgol
+  - tgorgol
```
Comment (pkosiec) [inline components/compass-runtime-agent/OWNERS]: Please also add `dbadura`. Thanks!

File: resources/compass/OWNERS [hunk 6]
```diff
@@ -0,0 +1,12 @@
+approvers:
+  - PK85
+  - aszecowka
+  - crabtree
+  - pkosiec
```
Comment (pkosiec) [inline resources/compass/OWNERS]: Please also add `dbadura`.

File: tests/compass-runtime-agent/OWNERS [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+  - janmedrek 
+  - Szymongib 
+  - franpog859 
+  - Maladie
+  - crabtree
+
+reviewers:
+  - PK85
+  - aszecowka
+  - pkosiec
```
Comment (pkosiec) [inline tests/compass-runtime-agent/OWNERS]: Please also add `dbadura`.

File: components/compass-runtime-agent/OWNERS (hunk 5 above)
Comment (majakurcius) [reply components/compass-runtime-agent/OWNERS]:
```suggestion
  - tgorgol
  - dbadura
```

Comment (majakurcius) [reply components/compass-runtime-agent/OWNERS]: Done.

File: resources/compass/OWNERS (hunk 6 above)
Comment (majakurcius) [reply resources/compass/OWNERS]: Done.

File: tests/compass-runtime-agent/OWNERS (hunk 7 above)
Comment (majakurcius) [reply tests/compass-runtime-agent/OWNERS]: Done.

**Pull request merged**